communication/README.md

```diff
@@ -28,7 +28,7 @@ groups/[SIGs] page.
 
 You can actively or passively participate in one of the following ways:
 - The community groups public meeting(s) listed in the above community groups page
-- Every Thursday at our [weekly community meeting] over [zoom] at [10am US Pacific Time]
+- Every Third Thursday at our [weekly community meeting] over [zoom] at [10am US Pacific Time]
```

Comment:
```suggestion
- Every Third Thursday at our [monthly community meeting] over [zoom] at [10am US Pacific Time]
```
Small little nit, but if you don't mind taking care of it in this pass it'd be great 🙏 

The link itself will also need to be fixed at the bottom of the doc :x

Reply:
Perfect, happy to! Thanks @mrbobbytables